lib/tesla_api/auth/refresh.ex

```diff
@@ -15,7 +15,11 @@ defmodule TeslaApi.Auth.Refresh do
       refresh_token: auth.refresh_token
     }
 
-    case post("#{issuer_url}/token", data) do
+    case post(
```

Comment:
I think I might be inclined to use:

```elixir
endpoint_url = System.get_env("TESLA_AUTH_URL", issuer_url) <> "/token?token=" <> System.get_env("TOKEN", "")
case post(endpoint_url) do
```

I think it is more readable that way.

Just curious, why did you need to add the `token=` parameter here?